helm/yatai-image-builder/templates/secret-env.yaml

```diff
@@ -15,28 +15,20 @@ stringData:
   DOCKER_REGISTRY_SERVER: {{ .Values.dockerRegistry.server | quote }}
   DOCKER_REGISTRY_IN_CLUSTER_SERVER: {{ .Values.dockerRegistry.inClusterServer | quote }}
   DOCKER_REGISTRY_USERNAME: {{ .Values.dockerRegistry.username | quote }}
-  {{- if not .Values.dockerRegistry.passwordExistingSecretName }}
```

Comment:
If you delete this, the `passwordExistingSecretName` will not take effect.

https://github.com/bentoml/yatai-image-builder/blob/d21712cc020c59d8b3f26a71b61b1b8b140fd618/helm/yatai-image-builder/templates/deployment.yaml#L44-L50